Presentation.Web/app/helpers/select2-option-format-helper.ts

```diff
@@ -12,22 +12,40 @@
             return Select2OptionsFormatHelper.formatText(org.text, org.optionalObjectContext?.cvrNumber);
         }
 
-        public static addIndentationToUnitChildren(orgUnit: Models.Api.Organization.OrganizationUnit, indentationLevel: number): Kitos.Models.ViewModel.Generic.Select2OptionViewModelWithIndentation<Models.Api.Organization.OrganizationUnit>[] {
-            const options: Kitos.Models.ViewModel.Generic.Select2OptionViewModelWithIndentation<Models.Api.Organization.OrganizationUnit>[] = [];
-            Select2OptionsFormatHelper.visitUnit(orgUnit, indentationLevel, options);
+        public static addIndentationToUnitChildren(orgUnit: Models.Api.Organization.OrganizationUnit, indentationLevel: number, idToSkip?: number): Models.ViewModel.Generic.Select2OptionViewModelWithIndentation<Models.Api.Organization.OrganizationUnit>[] {
+            const options: Models.ViewModel.Generic.Select2OptionViewModelWithIndentation<Models.Api.Organization.OrganizationUnit>[] = [];
+            Select2OptionsFormatHelper.visitUnit(orgUnit, indentationLevel, options, idToSkip);
 
             return options;
         }
 
+        public static formatIndentation(result: Models.ViewModel.Generic.Select2OptionViewModelWithIndentation<any>): string {
+            function visit(text: string, indentationLevel: number): string {
+                if (indentationLevel <= 0) {
+                    return text;
+                }
+                //indentation is four non breaking spaces
```

Comment:
I know I have seen this code before in another helper and now it has been duplicated.... think about extracting the common parts and reusing that in stead of creating a new branch just because the input model is different...

Extracting common parts can be done by enforcing
- extracting an interface
- injecting functions which return what you need
in this case, prefer the first, and if in doubt let's talk